cmd/copy.go

```diff
@@ -529,12 +561,12 @@ type cookedCopyCmdArgs struct {
 	excludeFileAttributes []string
 
 	// filters from flags
-	listOfFilesToCopy   []string
-	listOfFilesLocation string
-	recursive           bool
-	followSymlinks      bool
-	withSnapshots       bool
-	forceWrite          bool
+	listOfFilesToCopy  []string
+	listOfFilesChannel *chan string // We make it a pointer so we can check if it exists w/o reading from it & tack things onto it if necessary.
```

Comment:
Minor: IIRC `chan string` is nillable (the non-pointer version).  For that reason its not normal, AFAICT, to use pointers to channels https://stackoverflow.com/a/44367183

Reply:
Oh! I actually had no clue, I assumed it'd just initialize a bufferless channel.

Reply:
Corrected this behavior.